src/flattenizer.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Flattens an object
+ *
+ * @param {Object} obj - the object to unflatten
```

Comment:
Specify that this is a public function with `@public` and `@function`. Can leave off the name, though. JSDoc will grab it.